src/databricks/labs/ucx/runtime.py

```diff
@@ -187,6 +188,14 @@ def crawl_mounts(cfg: WorkspaceConfig):
     mounts.inventorize_mounts()
 
 
+@task("assessment", depends_on=[setup_schema])
+def enable_hms_lineage(cfg: WorkspaceConfig):
```

Comment:
It'll be a better fit if the init script is rolled out in the installer. It'll be simpler to support. Ask it in the installation flow as a question

Reply:
added in install.py